airgun/views/rhai.py

```diff
@@ -10,7 +10,28 @@
 )
 
 from airgun.views.common import BaseLoggedInView
-from airgun.widgets import ActionsDropdown, SatTable
+from airgun.widgets import (
+    ActionsDropdown,
+    SatTable,
+    GenericRemovableWidgetItem,
+    RemovableWidgetsItemsListView,
+)
+
+
+class InsightsOrganizationErrorView(BaseLoggedInView):
+    """View displayed when no Organization is selected or when the current organization has no
+    manifest
+    """
+    title = Text("//article[@id='content']/section/h1")
+    message = Text("//article[@id='content']/section/p")
+
+    @property
+    def is_displayed(self):
+        return (self.browser.wait_for_element(self.title, exception=False) is not None
+                and self.browser.wait_for_element(self.message, exception=False) is not None)
+
+    def read(self, widget_names=None):
+        return '{0}: {1}'.format(self.title.read(), self.message.read())
```

Comment:
I'm not sure we should mess with custom read, especially as there's no real need to - both title and message will be returned correctly with default read implementation.

Reply:
This really has no importance, is used in one place to get that error message, if we do not pack it here , we will pack else where, but really do not see, why we should have them separated, it is an error message.